packages/flutter_tools/test/commands.shard/hermetic/build_windows_test.dart

```diff
@@ -85,6 +85,8 @@ void main() {
         r'build\windows',
         '-G',
         generator,
+		'-A',
```

Comment:
please use spaces not tabs, and align with the previous lines

Reply:
Changed my editor